tools/topology2/sof-cnl-nocodec.conf

```diff
@@ -0,0 +1,71 @@
+
+#
+# Simple Machine - High level topology - Maps to machine driver.
+#
+#
+# PCM 0 <-> buffer1.0 <-> volume1.0 <-> buffer1.1 -> SSP0
+#
+
+<include/common/tokens.conf>
+<include/platform/intel/bxt.conf>
+<include/pipelines/pipeline-volume-playback.conf>
+<include/pipelines/pipeline-volume-capture.conf>
+<include/common/connection.conf>
+<include/common/endpoint.conf>
+<include/dais/ssp.conf>
+
+#
+# Pipeline definitions
+#
+
+# Pipeline ID:1 PCM ID: 0
+Object.pipeline-volume-playback."1.0" {
+	pcm_name	"Port0"
```

Comment:
We probably want to give this a more meaningful name than Port0 ?

Reply:
should this be SAI0 / SSP1, etc?

Reply:
@dbaluta this can literally be anything. On a real device for example, this would be "Headset", "Speaker" etc. But for nocodec, its simply SSP Port 0